packages/internals/src/__tests__/getSchema.test.ts

```diff
@@ -76,7 +89,9 @@ it('reads relative schema path from the nearest package.json', async () => {
 it('reads schema path from the nearest package.json and throws if path does not exist', async () => {
   const res = await testSchemaPath('pkg-json-invalid-path')
 
-  expect(res).toMatchInlineSnapshot(`[Error: Provided schema path \`wrong-path\` from \`package.json\` doesn't exist.]`)
+  expect(res).toMatchInlineSnapshot(
+    `[Error: Could not load schema from \`wrong-path\` provided by "prisma.schema" config of \`package.json\`: file or directory not found]`,
```

Comment:
New error message when explicitly configured in `pakcage.json`  schema not found